TimezoneSelect/timezone-options.tsx

```diff
@@ -0,0 +1,110 @@
+export const timeZoneOptions = [
+  {label: '(UTC-12:00) International Date Line West', value: 'Dateline Standard Time'},
```

Comment:
It looks that we should use the `utc[0]` value from [the initial data structure](https://github.com/ottofeller/kutir/blob/3b38e2d4fa3c2800bbce4841b1bf042c72f4c9c4/frontend/components/common/helpers/common/timezones.json#L71) as `option.value`. Otherwise it is not recognised by `moment()`:

<img width="780" alt="image" src="https://user-images.githubusercontent.com/58164/87657343-ffc05400-c75a-11ea-8d2c-7f0a05dc3a06.png">

Reply:
How did you get access to moment in console?

Reply:
@AleksandrIvanenko the dev console on their website works this way.